s3/pkg/meta/types/object.go

```diff
@@ -223,3 +223,12 @@ func (o *Object) GetCreateSql() (string, []interface{}) {
 
 	return sql, args
 }
+
+func (o *Object) GetUpdateMetaSql() (string, []interface{}) {
+	version := math.MaxUint64 - uint64(o.LastModified)
+	attrs, _ := json.Marshal(o.CustomAttributes)
+	sql := "update objects set customattributes =? where bucketname=? and name=? and version=?"
```

Comment:
From objectcopy.go in api, it seems user can update customattritubes, storage class, and content type, but here, only customattritubes will updated. Is that a bug?

Reply:
@sfzeng , Thanks for your reviewing, this is a bug. I"ll fix it in next commit

Reply:
acl and content type are updated in latest commit.  storage class have not been updated, because there is no storage class in db schema. I add a issue to trace it